Tasks/DotnetCLIRestore/dotnetrestore.ts

```diff
@@ -0,0 +1,42 @@
+import tl = require("azure-pipelines-task-lib/task");
+import tr = require("azure-pipelines-task-lib/toolrunner");
+import path = require("path");
+
+import * as restoreCommand from './restorecommand';
+
+let MessagePrinted = false;
+
+export class dotNetExe {
+
+    constructor() {
+    }
+
+    public async execute() {
+        tl.setResourcePath(path.join(__dirname, "task.json"));
+        this.setConsoleCodePage();
+
+        try {
+            await restoreCommand.run();
+        }
+        finally {
+            if (!MessagePrinted) {
+               console.log(tl.loc('NetCore3Update'));
```

Comment:
I don't think it makes sense to print this message as we are internally deciding .net core version.

Reply:
I copied this from CLIV2 task. If you think it shouldn't be displayed, I will remove it. Let me know.